recipes/gflags/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "2.1.2" %}
+
+package:
+  name: gflags
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/gflags/gflags/archive/v{{ version }}.tar.gz
+  sha1: 8bdbade9d041339dc14b4ab426e2354a5af38478
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
```

Comment:
Think we need to include Python in our requirements to make sure it links against the right MSVC.